nvme.c

```diff
@@ -6473,7 +6477,11 @@ static int dsm(int argc, char **argv, struct command *cmd, struct plugin *plugin
 		.timeout	= NVME_DEFAULT_IOCTL_TIMEOUT,
 		.result		= NULL,
 	};
+	gettimeofday(&start_time, NULL);
 	err = nvme_dsm(&args);
+	gettimeofday(&end_time, NULL);
+	if (cfg.latency)
+		printf(" latency: %s: %llu us\n", __func__, elapsed_utime(start_time, end_time));
```

Comment:
It may be more desirable to call `gettimeofday()` only when it's needed.
```
	if (cfg.latency)
		gettimeofday(&start_time, NULL);
	err = nvme_dsm(&args);
	if (cfg.latency) {
		gettimeofday(&end_time, NULL);
		printf(" latency: %s: %llu us\n", __func__, elapsed_utime(start_time, end_time));
	}
```

Reply:
Thanks for your comment. Just fixed as mentioned.  By the way both `submit_io()` and `passthru()` also may be needed to fix as same.